src/app.ts

```
@@ -1,29 +1,112 @@
+    if (!isValid) {
+      return c.json({ error: "Invalid signature" }, 401);
+    }
+    await upvoteJobBatch(c);
+    logger.info("Upvote job batch complete");
+    return c.json(
+      {
+        success: true,
+        message: "Webhook received and verified",
+      },
+      200
+    );
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)